### PR TITLE
Add prometheus repository

### PR DESCRIPTION
**Pull request by huavanthong**
Hi Thomas,

I have updated the configuration for the prometheus repository to fix the error the pipeline.
Please help me merge it. 

Thank you and best regards,
Thong Hua

### CODE REVIEW

File: config/robotframework_aio/release_items_Robotframework_AIO.json
Comment (test-fullautomation) [inline config/robotframework_aio/release_items_Robotframework_AIO.json:1]: Hi Thong,

this must be part of "0.12.1.;1.13.0." - entry as yesterday written.
Please correct.

Thank you,
 Thomas

**Review by test-fullautomation — CHANGES_REQUESTED** [review]
Hi Thong,
please see my finding.
Thank you,
 Thomas

File: config/robotframework_aio/release_items_Robotframework_AIO.json
Comment (huavanthong) [reply config/robotframework_aio/release_items_Robotframework_AIO.json:1]: Hi Thomas, 

I have updated it as your comment.
If I have any mistakes, please let me know.

Thank you,
Thong Hua

Comment (test-fullautomation) [reply config/robotframework_aio/release_items_Robotframework_AIO.json:1]: Hi @huavanthong ,

this is unfortunately still wrong.

As written above you  need to make:
 "0.12.1.;0.13.0."

This makes that the info will be visible for intermediate release 0.12.1. and major 0.13.0.
All intermediate version changes need to be visible for this specific intermediate version as well as next major release.

Thank you,
 Thomas

**Review by test-fullautomation — CHANGES_REQUESTED** [review]
Hi Thong,
please check my finding.
Thank you,
 Thomas

**Comment by huavanthong** [comment]
Thank you for your pointing. I have updated it.

File: config/robotframework_aio/release_items_Robotframework_AIO.json
Comment (test-fullautomation) [reply config/robotframework_aio/release_items_Robotframework_AIO.json:1]: Hi @huavanthong ,
better :-)
One "." dot in front of the semicolon is still missing.
This message would be displayed e.g. for 0.12.10, too. But it should be only displayed for 0.12.1. versions.
Thank you,
 Thomas

**Review by test-fullautomation — CHANGES_REQUESTED** [review]
Hi Thong,
please check my finding.
Thank you,
 Thomas

File: config/robotframework_aio/release_items_Robotframework_AIO.json
Comment (huavanthong) [reply config/robotframework_aio/release_items_Robotframework_AIO.json:1]: Hi Thomas,

I'm sorry for these mistakes. I have updated it and gained a better understanding.
Beside that, I also tested it on my machine. It worked 👍  , all the release info are combined together.

Thank you for your support.

Best regards,
Thong Hua

**Pull request merged**